app/utils/authUtils.js

```
@@ -0,0 +1,101 @@
+export const removeStoredToken = () => {
+  if (typeof window === 'undefined') return;
+  localStorage.removeItem(TOKEN_KEY);
+  localStorage.removeItem(USER_KEY);
+};
+
+/**
+ * Get stored user from localStorage
+ * @returns {object|null} The user object or null
+ */
+export const getStoredUser = () => {
+  if (typeof window === 'undefined') return null;
```

ESLint

[error] Expected { after 'if' condition.

(curly)